src/DynamoCoreWpf/Views/Core/NoteView.xaml.cs

```diff
@@ -84,8 +85,19 @@ void OnNoteTextPreviewMouseDown(object sender, MouseButtonEventArgs e)
 
         private void OnEditItemClick(object sender, RoutedEventArgs e)
         {
+
+            var currentZoom = ViewModel.WorkspaceViewModel.Zoom;
+            if (currentZoom > MINIMUM_ZOOM_DIRECT_NODE_EDIT)
```

Comment:
maybe just use `ViewModel.WorkspaceViewModel.Zoom` directly, dosent look like you are using the currentZoom variable anywhere else?